disttask/framework/dispatcher/dispatcher_pool.go

```diff
@@ -110,54 +111,56 @@ type dispatcher struct {
 	detectPendingGTaskCh chan *proto.Task
 }
 
-// NewDispatcher creates a dispatcher struct.
-func NewDispatcher(ctx context.Context, taskTable *storage.TaskManager) (Dispatch, error) {
-	dispatcher := &dispatcher{
+// NewPool creates a Pool struct.
```

Comment:
I think we needn't change this name to xxPool. 
Feel general xxPool, after the creation can have getXX, putXX and other interfaces. I don't think it's necessary here.